src/sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import { Tooltip, OverlayTrigger } from "react-bootstrap";
+import "./sidebar.css";
+
+import image from "../assets/images/AnitaBLogo.png";
+
+const Sidebar = (props) => {
+  const [isNotActive, setNotActive] = useState("true");
+  const [isDropdownActive, setDropdownActive] = useState("false");
+  var barsIcon = <i className="fas fa-bars"></i>;
+  var crossIcon = <i className="fas fa-times-circle"></i>;
+  return (
+    <div>
+      <div className="wrapper">
+        <nav id="sidebar" className={isNotActive ? "active" : ""}>
+          <button
+            type="button"
+            id="sidebarCollapse"
+            onClick={() => setNotActive(!isNotActive)}
+            className="btn btn-custom"
+          >
+            <span className={isNotActive ? "" : "hidden"}>{barsIcon}</span>
+            <span className={isNotActive ? "hidden" : ""}>{crossIcon}</span>
+          </button>
+          <div className="sidebar-header">
+            <img
+              alt="user_logo"
+              src={image}
+              className="rounded-circle usr-image"
+              height={isNotActive ? "20" : "70"}
+              width={isNotActive ? "20" : "70"}
+            ></img>
+            <h3>User Name</h3>
```

Comment:
Here you can use the `user` data as it was used in `Navigation.jsx` line 117-123 to show the actual user's name
<img width="935" alt="Screen Shot 2021-06-18 at 11 37 09 am" src="https://user-images.githubusercontent.com/29667122/122492666-86e01100-d029-11eb-90e0-0714db269e84.png">